.gitignore

```diff
@@ -13,3 +13,4 @@ cli/build/
 cli/dist/
 .vscode/
 badgeyay/
+.coverage
```

Comment:
what is `.coverage` @yashLadha  ?
:sweat_smile:

Reply:
When we do the coverage test in the local machine, this is generated. The same tests tests that happen in travis. @gabru-md

Reply:
okay. Thanks :D